core/src/main/java/io/undertow/server/protocol/http2/Http2UpgradeHandler.java

```diff
@@ -76,7 +76,7 @@ public void handleRequest(HttpServerExchange exchange) throws Exception {
         final String settings = exchange.getRequestHeaders().getFirst("HTTP2-Settings");
         if(settings != null && upgrade != null
                 && upgradeStrings.contains(upgrade)) {
-            if(HttpContinue.requiresContinueResponse(exchange) && false) {
+            if(HttpContinue.requiresContinueResponse(exchange) && false) { //NOSONAR https://issues.redhat.com/browse/UNDERTOW-2135
```

Comment:
the sole point is that this code is changed because UNDERTOW-2135 is solved.

Reply:
When I first added the NOSONAR tag, 2135 was still open -- and the fix was to delete this block.

Fix in Master has been merged, committed and pushed.

Reply:
@gorky exactly! Thanks for updating the PR so quickly